OpenRA.Mods.Common/Traits/Husk.cs

```diff
@@ -183,4 +196,41 @@ public class HuskSpeedInit : ValueActorInit<int>, ISingleInstanceInit
 		public HuskSpeedInit(int value)
 			: base(value) { }
 	}
+
+	public class DragAndCrush : Activity
+	{
+		readonly LocomotorInfo info;
+
+		public DragAndCrush(Actor self, LocomotorInfo info, int dragSpeed, WPos finalPosition)
+		{
+			this.info = info;
+
+			CrushAction(self, finalPosition, (notifyCrushed) => notifyCrushed.WarnCrush);
```

Comment:
I added a warning. But now thinking more about it, the actor should have already given a warning while alive. So now you effectively give 2 warnings which increases survival rates

Reply:
So this should be removed again? I don't see a reason to give two warnings.

Reply:
Removed